src/pages/_app.tsx

```diff
@@ -4,13 +4,16 @@ import { ChakraProvider } from "@chakra-ui/react";
 import { SessionProvider } from "next-auth/react";
 import type { AppProps } from "next/app";
 
+import { BodyContent } from "../components/BodyContent";
 import { theme } from "../styles/theme";
 
 function MyApp({ Component, pageProps: { session, ...pageProps } }: AppProps) {
     return (
         <SessionProvider session={session}>
             <ChakraProvider theme={theme}>
-                <Component {...pageProps} />
+                <BodyContent>
+                    <Component {...pageProps} />
+                </BodyContent>
```

Comment:
Envolvendo toda as as páginas  com  o BodyContent faz com que a página de login também tenha. Dá pra fazer uma condição lá no componente para ele se comportar de outra forma se a página ativa for a de login.